connectors/servicenow/src/test/resources/servicenow/responses/success/cards/card_xx.json

```diff
@@ -10,32 +10,32 @@
       "header": {
         "title": "[SERVICE NOW] APPROVAL REQUEST",
         "subtitle": [
-          "REQ0010002"
+          "REQ0010003"
```

Comment:
Is the allowingAnyArrayOrdering not working?

Reply:
An oversight. Corrected.